Secway sounds, gas mask sprite revert.

Weisyl opened the pull request:
Secways now have sounds for whenever they move, gas masks use their old
sprites once again, the AK-922, G17, and Abzats can now spawn via summon
guns (as they should've been before), and there are a few more options
in the Select Equipment menu for admins.

Weisyl committed:
Secway sounds, gas mask sprite revert.

Gas masks now have selectable sprites, the AK-922, G17, and Abzats can spawn in summon guns,
camo boots have a very slightly edited on-mob sprite.

AliiTea commented on code/modules/mob/living/simple_animal/corpse.dm:
```diff
@@ -154,4 +154,18 @@
+	corpsehelmet = /obj/item/clothing/head/ushanka
+
+
+/obj/effect/landmark/mobcorpse/russian/ranged/spetsnaz
```
They are used for Away Mission shit.. and whenever you want to place a dead clwune

Weisyl replied:
It's for a new Russian spetsnaz mob using pre-existing gear, won't spawn anywhere by default at the moment, might be useful for an away mission though.

AliiTea replied:
I will

AliiTea commented on code/modules/clothing/masks/gasmask.dm:
```diff
@@ -1,11 +1,11 @@
 /obj/item/clothing/mask/gas
```
Kek why not just remove the newer sprites and rename this sprite?

Edit:

Fuck ya spooky Chaplain time!

Weisyl replied:
I suggested that but I was told to simply revert on the sprite anyway.

AliiTea replied:
This would be a lot better

Weisyl replied:
I've made another commit / PR that now has it as a selection system.

AliiTea commented on code/modules/vehicles/segway.dm:
```diff
@@ -5,6 +5,11 @@
 	icon_state = "secway"
 	callme = "secway"
 	anchored = 0 //This is not so heavy
+	move_speed = 12
```
This seems a bit ridiculous..

Weisyl replied:
That was a bit of experimentation with attempting to make it slower which I forgot to remove. Dunno if it makes this a DNM PR though as it didn't seem to affect anything.

Weisyl commented on code/modules/vehicles/segway.dm:
```diff
@@ -6,6 +6,10 @@
 	anchored = 0 //This is not so heavy
 
+	Move()
+		playsound(src.loc, 'sound/effects/secway.ogg', 75, 1)
```
No, but I cut it down enough that it wasn't awfully fucky and it didn't seem to cut out other sounds.

Weisyl committed:
Possible merge conflicts fix

This might fix it, at this rate I frankly have no idea and I have to go get some other shit done.
